libr/syscall/d/linux-x86-32.sdb.txt

```diff
@@ -363,4 +363,4 @@ pkey_free=0x80,382,1,
 statx=0x80,383,5,
 arch_prctl=0x80,384,2,
 io_pgetevents=0x80,385,6,
-rseq=0x80,386,4
+rseq=0x80,386,4,
```

Comment:
Not having this trailing comma shouldnt be abreason to crash or not to work

Reply:
I know, indeed I fixed the reason of the crash (replaced `< 3` with `< 4` but since this is the only entry that is not compliant with the rest of the entries, I also fixed it as well.